cloud-annotations/src/main/java/cloud/commandframework/annotations/MethodCommandExecutionHandler.java

```diff
@@ -63,7 +63,7 @@
     }
 
     @Override
-    public void execute(final @NonNull CommandContext<C> commandContext) {
+    public void execute(final @NonNull CommandContext<C> commandContext) throws Throwable {
```

Comment:
Throwable sounds bad. This includes Error, which explicitly shouldn't ever be caught.